wordpress-mobile/WordPress-FluxC-Android
# Woo: Issue/1402 cache jetpack user email to SiteModel

Fixes #1402 by adding the `jetpack_user_email` param to the xmlrpc `wp.getOptions` endpoint and storing locally.

### Testing
- Click on the `Sign in` button in the example app.
- Enter username, password and xmlrpc url for the site with Jetpack installed/activated/connected.
- Once successfully logged in, click on `Sites -> Fetch first site`.
- Verify that the xmlrpc `wp.getOptions` endpoint is called and the `jetpack_user_email` corresponding to the login credentials is received correctly.
- Login with a different jetpack connected user for the same site and verify that the `jetpack_user_email` is changed accordingly.

Note that the `jetpack_user_email` field would only be available in the API response if jetpack is installed, activated and connected.
